lib/Pusher.php

```diff
@@ -330,6 +352,7 @@ public function trigger( $channels, $event, $data, $socket_id = null, $debug = f
 
 		if ( $socket_id !== null )
 		{
+			$this->validate_socket_id( $socket_id );
```

Comment:
To make it more clear, I will end up with this situation:

```php
		if ( $socket_id !== null )
		{
			$this->validate_socket_id( $socket_id );
		}

...

		if ( $socket_id !== null )
		{
			$post_params[ 'socket_id' ] = $socket_id;
		}
```

which means duplication. It depends if you think this is worth it in order for it to fail asap.